phpcs.xml.dist

```diff
@@ -6,7 +6,58 @@
 
   <exclude-pattern>*/Test/*</exclude-pattern>
 
-  <rule ref="PHPCS" />
+  <!-- The following comes from the 2.9 PHPCS ruleset.xml { -->
+
+  <!-- Include the whole PEAR standard -->
+  <rule ref="PEAR"/>
```

Comment:
Did the PHPCS coding standards change dramatically upstream?

Reply:
yes 😄 it used to have `PHPCS` as a ruleset which has now been totally removed. This was raised as an issue in the phpcs issue queue and their "solution" (copy and paste in the rules from the previous ruleset) is the one I followed.

Reply:
I see! This is a good solution for now, it keeps the disruption to a minimum.